src/test/codegen/mem-replace-direct-memcpy.rs

```diff
@@ -17,8 +17,8 @@ pub fn replace_byte(dst: &mut u8, src: u8) -> u8 {
 
 // CHECK-NOT: call void @llvm.memcpy
 // CHECK: ; core::mem::replace
-// CHECK-NOT: call void @llvm.memcpy
```

Comment:
What happened here? These should not be changed

Reply:
The test was failing, changed it again, but I am not sure why this test is getting affected by this code

Reply:
I don't think it is. Try reverting all the changes to this file

Reply:
okay I reverted everything..